doc/manuals.jp/admin/database_model.md

```diff
@@ -251,11 +251,11 @@ Orion Context Broker は、データベース内で次のサブセクション
 -   **maxFailsLimit**: 接続試行の最大制限を指定するために使用されるオプションのフィールド。これにより、失敗した通知の数に達すると、サブスクリプションは自動的に非アクティブ状態に移行します
 -   **failsCounter**: サブスクリプションに関連付けられた連続して失敗した通知の数。これは、通知の試行が失敗するたびに1つずつ増加します。通知の試行が成功すると、0にリセットされます
 -   **altTypes**: サブスクリプションに関連付けられた変更タイプ (alteration types) のリストを含む配列。フィールドが含まれていない場合は、デフォルトが想定されます ([このドキュメント](../orion-api.md#subscriptions-based-in-alteration-type)を確認してください)
-
 -   **covered**: すべての `attrs` を通知に含める必要があるか (値がtrueの場合)、トリガー・エンティティに存在するものだけを含める必要があるか
     (値がfalseの場合、またはフィールドが省略されている場合) を指定するブール・フィールド
     詳細については、Orion API 仕様の[対象サブスクリプション・セクション](../orion-api.md#covered-subscriptions)
     を参照してください。
+-   **notifyOnMetadataChange**: メタデータの変更に対してのみ通知をトリガーするように定義するブール変数 (値が true の場合) またはメタデータのみが変更された場合、通知のトリガーを回避する (値が false の場合)
```

Comment:
This was changed in  [3ba1ead](https://github.com/telefonicaid/fiware-orion/pull/4310/commits/3ba1ead3128937e80932ee1f3210bacc23f3d475) in PR #4310, just in case this need to be adjusted before merging...